### ISSUE TITLE
Animations as individual files

### ISSUE THREAD

**Issue by DrakeFruit** [sboxgame/issues#3930]
### For?

S&Box

### What can't you do?

When working with animations, they're baked into the model you want to use them on, this works if you're making a game, but becomes an issue for UGC. being able to use generic humanoid animations on any model with a humanoid skeleton would make them shareable on asset party, addon friendly, and easier to use overall imo.

### How would you like it to work?

create an animation file that contains the animation and it's parameters
play the same animation on any model with a humanoid skeleton

### What have you tried?

I have tried using bone merging / creating my own copy of the base model with my own animations for UGC, but it's just inconvenient and janky.

**Comment by TheBoxTrot** [comment]
Thats kinda what the whole base model system was for wasn't it? You can make a model with a bunch of animations and set the base model to the model you want to use

**Comment by MaxLebled** [comment]
Yup, the "base model" feature is there primarily to let you extend an existing model's animation set

**Comment by DrakeFruit** [comment]
that still

**Comment by DrakeFruit** [comment]
didn't mean to press enter or close it what.
anyways, I still want to be able to share animations / use animation files. them being embedded in models seems inconvenient

**Comment by DrakeFruit** [comment]
if you were to make an addon to add animations to a character, would you change the model the entity uses to one with your animations referencing the base? what if you don't have access to the base model / another addon also tries to do this

**Comment by TheBoxTrot** [comment]
if you upload a addon that contains a model that references another model that other model will also be uploaded I don't see the point in being able to upload animations when they are pretty much tied to models

**Comment by surv0013** [comment]
Without this feature it is impossible to make animations for weapons properly. For example, if you have a bunch of pistols in the game and you want to use the same animation for all of them, then you have to put this animation into each model. 😔

**Comment by DrakeFruit** [comment]
it's a bit hacky but you can have one vmdl with all your animations and reference that as a base model in the guns that use it
